.github/workflows/r-ci.yml

```diff
@@ -0,0 +1,39 @@
+# based on check-release from the r-lib examples
+name: R CI
+
+on: push
+
+jobs:
+  check:
+    runs-on: ubuntu-16.04
+    env:
+      RSPM: "https://packagemanager.rstudio.com/cran/__linux__/xenial/latest" 
+    steps:
```

Comment:
Maybe a space between the setup items and the `steps` just for clarity. Everything in steps can be squished together.